src/psyclone/tests/psyad/adjoint_visitor_test.py

```diff
@@ -781,6 +793,129 @@ def test_loop_logger(fortran_reader, caplog):
             "contains no active variables" in caplog.text)
 
 
+# AdjointVisitor.ifblock_node()
+
+def test_ifblock_node_active_error(fortran_reader):
+    '''Test that the ifblock_node method raises the expected exception
+    if no active variables are specified.
+
+    '''
+    tl_psyir = fortran_reader.psyir_from_source(TL_IF_CODE)
+    ifblock = tl_psyir.walk(IfBlock)[0]
+    adj_visitor = AdjointVisitor(["a", "b", "c"])
+    with pytest.raises(VisitorError) as info:
+        _ = adj_visitor.ifblock_node(IfBlock)
```

Comment:
I'm not sure how this even works but presumably you meant `ifblock` here rather than `IfBlock`?

Reply:
I did. I've not idea how the test worked either, but I've updated it.